meetings/agenda.md

```diff
@@ -10,3 +10,23 @@ Please file pull requests to add, or discuss items to add, to the agenda.
 ### Abandon `mezzio/mezzio-aurarouter`
 
 As per [the RFC](https://github.com/mezzio/mezzio-aurarouter/issues/12), should we abandon this package and archive it?
+
+### Maintenance Chores
+
+#### Old Release Branch Clean Up
+
+Packages like `laminas-validator` have a _lot_ of branches. Many of these branches will never see any more commits, and it is trivial to re-create those branches if we ever need to add a patch to an old release.
+
+Is it acceptable to delete release branches older than say 1 year?
```

Comment:
I can answer this: depends on the PHP version that they support :D

Reply:
> I can answer this: depends on the PHP version that they support :D

If you delete an old release branch, will that make old tags/releases from that branch uninstallable? :thinking:

Reply:
No, old tags stay there, but it is not possible to open a patch against an old (security-maintained) version, and then merge-up to latest.

For example, let's say that there are few branches:

* 1.20.x - supports PHP 8.0
* 1.21.x - supports PHP 8.0
* 1.22.x - supports PHP 8.1
* 1.23.x - supports PHP 8.1
* 1.24.x - supports PHP 8.1 - current mainline
* 2.0.x - unreleased

At the time of this writing, https://www.php.net/supported-versions.php says that:

 * PHP 8.0 is EOL
 * PHP 8.1 is maintained until 2026-01-01

This means that we can safely delete `1.20.x` and `1.21.x`, because both no longer accept any security patches by contributors or maintainers.

Reply:
Thanks! Might be worth a quick discussion at the August meet so everyone is on the same page then!

Reply:
release branches for older releases are only needed for security at which point they can be recreated from a tagged release. No TSC decision is needed on this as those branches must not have unreleased stuff on them by definition. Same for milestones

The only issue is that deleting those branches requires disabling branch protection.

Reply:
> The only issue is that deleting those branches requires disabling branch protection.

So, it's the kind of thing you'll need to do periodically and carefully… remembering to re-enable branch protection again when you're finished

Reply:
All tags exist in the latest release branch. When they do not it is a big big concern and must be investigated for potential tampering/security issue.
As for how git works tags are fully self sufficient. They do not require for any branch to have their commit in history.